lib/awful/titlebar.lua

```diff
@@ -453,6 +453,32 @@ local function get_titlebar_function(c, position)
     end
 end
 
+--- Call `request::titlebars` to allow themes or rc.lua to create them even
+-- when `titlebars_enabled` is not set in the rules.
+-- @tparam client c The client.
+-- @tparam[opt=false] boolean hide_all Hide all titlebars except `keep`
+-- @tparam string keep Keep the titlebar at this position
+-- @treturn boolean If the titlebars were loaded
+local function load_titlebars(c, hide_all, keep)
```

Comment:
This function is only ever called with `hide_all=true`. Why is this a parameter?

Reply:
A second PR will add `hide_all`/`show_all`/`toggle_all`. I didn't want to scope creep this PR.